yang/frr-route-types.yang

```diff
@@ -162,9 +162,7 @@ module frr-route-types {
   typedef ipv6-multicast-group-prefix {
     type inet:ipv6-prefix {
       pattern
-        '(((FF|ff)[0-9a-fA-F]{2}):)([0-9a-fA-F]{0,4}:){0,5}((([0-9a-fA-F]{0,4}:)?(:|[0-9a-fA-F]{0,4}))|(((25[0-5]|2[0-4][0-9]|[01]?[0-9]?[0-9])\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9]?[0-9])))(/((1[6-9])|([2-9][0-9])|(1[0-1][0-9])|(12[0-8])))';
-      pattern
-        '(([^:]+:){6}(([^:]+:[^:]+)|(.*\..*)))|((([^:]+:)*[^:]+)?::(([^:]+:)*[^:]+)?)(/.+)';
+	'(([fF]{2}[0-9a-fA-F]{2}):).*';
```

Comment:
I think we need an actual full regex here — your change accepts a lot of random invalid data...

Reply:
I have verified this, now we are accepting only valid ipv6 multicast address here. Can you specify the random invalid inputs which will get accepted here?

Reply:
```
ffF1:ds.dasdasdhlklklkl
```

Reply:
frr(config)# ipv6 pim rp 2::2 ffF1:ds.dasdasdhlklklkl
% Unknown command: ipv6 pim rp 2::2 ffF1:ds.dasdasdhlklklkl
frr(config)# 
This is taken care by the CLI but yeah in case FRR needs the full regex in the yang, I can update that. 
This regular expression is taken directly from the RFC 8294.

Reply:
> ffF1:ds.dasdasdhlklklkl

The above expression is taking care of this case as well.

YANG error(s):
 Invalid union value "ffF1:ds.dasdasdhlklklkl" - no matching subtype found.
 YANG path: Schema location /frr-routing:routing/control-plane-protocols/control-plane-protocol/frr-pim:pim/address-family/frr-pim-rp:rp/static-rp/rp-list/group-list-or-prefix-list/group-list/group-list.
frr(config)#

Reply:
Note: Since the field is of type ipv6-prefix, tha pattern matching at ipv6-prefix is also applicable here.

Reply:
> Note: Since the field is of type ipv6-prefix, tha pattern matching at ipv6-prefix is also applicable here.

Oh right I forgot about that